client/src/reducers/index.js

```diff
@@ -63,6 +64,7 @@ const Reducer = undoable(
     ["universe", universe],
     ["world", world],
     ["categoricalSelection", categoricalSelection],
+    ["continuousSelection", continuousSelection],
```

Comment:
This data structure seems like it could be simplified since the strings and variables are always the same.

Reply:
could you suggest something concrete?   I don't think we can assume that the functions and fields have the same name, but they often will, and I'm open to improving this.  I just couldn't come up with a concrete idea!

Reply:
I'm thinking on it. This should block the merge.